src/pybind/mgr/orchestrator.py

```diff
@@ -1042,12 +1050,15 @@ class PlacementSpec(object):
     """
     For APIs that need to specify a node subset
     """
-    def __init__(self, label=None, nodes=None):
+    def __init__(self,
+                 label=None,  # type: Optional[str]
+                 hosts=None  # type: Optional[List[str]]
```

Comment:
could you move s/nodes/hosts/ related changes into its own commit?

Reply:
You mean, I should first unify the mess of (nodes,hosts) to nodes and then rename everything back to hosts?

Reply:
why would you change them to nodes first?

Reply:
exactly. that's why I converged to `hosts`

Reply:
> I should first unify the mess of having both (nodes and hosts) to nodes and then rename everything back to hosts?

@sebastian-philipp i am confused. let me rephrase: i don't think you should first unify the mess of having both (nodes and hosts) to nodes and then rename everything back to hosts. instead, i think a commit which just do PlacementSpec  and s/nodes/hosts/ change would be good enough.

Reply:
:+1: